### PR TITLE
Linted

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

**Comment by dherrada** [comment]
Disabled duplicate code

File: adafruit_portalbase/__init__.py [hunk 1]
```diff
@@ -400,7 +400,9 @@ def _fill_text_labels(self, values):
                     string = func(values[value_index])
                 else:
                     try:
-                        string = "{:,d}".format(int(values[value_index]))
+                        string = "{:,d}".format(
+                            int(values[value_index])
+                        )  # pylint: disable=consider-using-f-string
```
Comment (kattni) [inline adafruit_portalbase/__init__.py:405]: Why is this needed? This is disabled in the .pre-commit-config.yml file.

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks! Please see below.

File: adafruit_portalbase/__init__.py (hunk 1 above)
Comment (dherrada) [reply adafruit_portalbase/__init__.py:405]: @kattni Oh hah yeah you're right. I need to figure out how to get vim to default to the repos pylintrc but I think it could have also had something to do with doing this at 3AM

**Review by kattni — APPROVED** [review]
